src/redux/initialState.js

```diff
@@ -1270,156 +1270,6 @@ const initialState = {
-      availability: 'in Stock',
-      newFurniture: true,
-    },
-    {
-      id: 'aenean-ru-bristique-72',
-      name: 'Aenean Ru Bristique 72',
-      category: 'bedroom',
-      price: 30,
-      oldPrice: '$40',
-      stars: 3,
-      promo: 'sale',
-      image:
-        'https://ksassets.timeincuk.net/wp/uploads/sites/56/2018/11/Take-a-tour-of-this-pretty-four-bedroom-Yorkshire-stone-detached-house-built-in-1921-bed.jpg',
-      description:
-        'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.',
-      availability: 'in Stock',
-      newFurniture: true,
-    },
-    {
-      id: 'aenean-ru-bristique-72',
-      name: 'Aenean Ru Bristique 72',
-      category: 'bedroom',
-      price: 30,
-      oldPrice: '$40',
-      stars: 3,
-      promo: 'sale',
-      image:
-        'https://cdn.home-designing.com/wp-content/uploads/2019/07/modern-and-retro-accessory-ideas-for-transitional-bedroom-design-1024x717.jpg',
-      description:
-        'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.',
-      availability: 'in Stock',
-      newFurniture: true,
-    },
-    {
-      id: 'aenean-ru-bristique-72',
-      name: 'Aenean Ru Bristique 72',
-      category: 'bedroom',
-      price: 30,
-      oldPrice: '$40',
-      stars: 3,
-      promo: 'sale',
-      image:
-        'https://hips.hearstapps.com/hmg-prod.s3.amazonaws.com/images/romantic-bedrooms-pickart-vanatta-217x-1578085880.jpg?crop=1.00xw:0.844xh;0,0.143xh&resize=640:*',
-      description:
-        'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.',
-      availability: 'in Stock',
-      newFurniture: true,
-    },
-    {
-      id: 'aenean-ru-bristique-72',
-      name: 'Aenean Ru Bristique 72',
-      category: 'bedroom',
-      price: 30,
-      oldPrice: '$40',
-      stars: 3,
-      promo: 'sale',
-      image: 'https://upload.wikimedia.org/wikipedia/commons/0/0d/Bedroom_Mitcham.jpg',
-      description:
-        'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.',
-      availability: 'in Stock',
-      newFurniture: true,
-    },
```

Comment:
Z czego wynika ta zmiana? Ona nie jest związana z zadaniem

Reply:
nie wiem, sluchaj wyskoczylo mi to do zmiany w bledach